install.sh

```diff
@@ -3872,6 +3874,10 @@ setup_hook_templates() {
         HOOK_NAMES="$MANAGED_HOOK_NAMES"
     fi
 
+    # The template target directory exists,
+    # make sure the added paths (e.g. `hooks`) do too
+    mkdir -p "$TARGET_TEMPLATE_DIR" >/dev/null 2>&1
+
```

Comment:
Thats where I create the directory. The comment should be: "The template target directory partially exists, make sure the added parts (e.g. hooks) do too...